kolibri/plugins/html5_app_renderer/assets/src/views/index.vue

```diff
@@ -83,6 +83,8 @@
     height: 100vh
     max-height: calc(100vh - 24em)
     min-height: 400px
+    overflow-x: auto
+    -webkit-overflow-scrolling: touch
```

Comment:
Can do without `-webkit-overflow-scrolling`, as it appears to have almost no support https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling